samples/model/my-model.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: Model
+metadata:
+  name: <YOUR MODEL NAME>
+spec:
+  modelName: <YOUR MODEL NAME>
+  primaryContainer:
+    containerHostname: xgboost
+    # The source of the model data
+    modelDataURL: s3://<YOUR BUCKET>/path
```

Comment:
should we wrap S3 paths like this `s3://<YOUR BUCKET/PATH>` ?

Reply:
Helps to imply you need to change path as well.  Sounds good